pkgs/development/tools/asn1ate/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, python, pythonPackages, buildPythonPackage, self }:
```

Comment:
pythonPackages has python and buildPythonPackage as attributes. Could you remove those two here and use either a `let` expression or `with` instead.